local.mk

```diff
@@ -1,5 +1,5 @@
 ifeq ($(MAKECMDGOALS), dist)
-  dist-files += $(shell git ls-files)
+  dist-files += $(shell cat dist-files)
```

Comment:
You're right. My initial thought was just generating dist-files initially just committing it to the repository. But that would require every new file to be added to that list; that would be confusing and hard to remember in my opinion.